opensearchpy/client/plugins.py

```diff
@@ -45,9 +44,7 @@ def _dynamic_lookup(self, client):
                 setattr(client, plugin, getattr(self, plugin))
             else:
                 warnings.warn(
-                    "Cannot load `{plugin}` directly to OpenSearch. `{plugin}` already exists in OpenSearch. Please use `OpenSearch.plugin.{plugin}` instead.".format(
-                        plugin=plugin
-                    ),
+                    f"Cannot load `{plugin}` directly to AsyncOpenSearch. `{plugin}` already exists in AsyncOpenSearch. Please use `AsyncOpenSearch.plugin.{plugin}` instead.",
```

Comment:
The generator has replacing **AsyncOpenSearch** with **OpenSearch** code, but it's not happening, and I'm not sure why. https://github.com/saimedhi/opensearch-py/blob/main/utils/generate-api.py#L655

Reply:
I'll look.